suite-native/app/app.config.ts

```diff
@@ -45,7 +51,7 @@ const projectIds = {
 export default ({ config }: ConfigContext): ExpoConfig => {
     const buildType = (process.env.ENVIRONMENT_EXPO as BuildType) ?? 'debug';
     const name = appNames[buildType];
-    const slug = appNames[buildType].toLowerCase().split(' ').join('-');
+    const slug = appSlugs[buildType].toLowerCase().split(' ').join('-');
```

Comment:
I think you don;t need to do this magic with lowercase, split and join, just write it in correct format in appSlugs since it's used only there

Reply:
Yeah, good point. Before it was derived from appNames, but now doesn't make sense. Fixed